pkgs/applications/audio/VoiceOfFaust/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, pkgs, callPackage, fetchFromGitHub, faust2jack, helmholtz, mrpeach }:
+stdenv.mkDerivation rec {
+  name = "VoiceOfFaust-${version}";
+  version = "v0.7";
```

Comment:
Better move the "v" to the rev attribute of `fetchFromGitHub`, as versions should start with a digit. See http://nixos.org/nixpkgs/manual/#idm140737319248160